### ISSUE TITLE
Secure links for 3rd party providers

### ISSUE THREAD

**Issue by guardrex** [aspnet/Docs#8803]
Work this issue with related issue: **Cross-link logging providers to Logging topic** #8804

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* Version Independent ID: 726e3bf1-f367-d733-8933-bccc04da0e16
* Content: [Logging in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-2.1)
* Content Source: [aspnetcore/fundamentals/logging/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/logging/index.md)
* Product: **aspnet-core**
* GitHub Login: @ardalis
* Microsoft Alias: **tdykstra**

**Comment by guardrex** [comment]
@Rick-Anderson @scottaddie Given https://github.com/aspnet/Docs/pull/8802#issuecomment-426381583 ...

> For my team there's no "maturity test" - I just check that the links work and seem to do what they claim to do. It's not even a list of recommended providers at https://github.com/aspnet/Logging/blob/master/README.md, just a list of providers that exist.

I'd like to add a line to the end of the 3rd party providers section that expresses that sentiment. Below, I'm showing it added to the last paragraph, which is currently a single sentence. I make a word change there, too ("framework" :point_right: "provider") ...

> For more information, see each framework's documentation. 

... to ...

> For more information, see each provider's documentation. Third-party logging providers aren't maintained or supported by Microsoft or the .NET Foundation.

:point_up: Perhaps, it won't be those words exactly, but something along those lines. Let me know what you want to do, if anything.

Otherwise, this issue is just about updating the links (everywhere) to their secure endpoints.

**Comment by scottaddie** [comment]
@guardrex In your suggested text, remove "or the .NET Foundation".

**Comment by scottaddie** [comment]
Let's just stick with "supported".

**Comment by guardrex** [comment]
@tdykstra You might want this one, too, for your Logging topic updates.

TL;DR It's a simple change. In the Logging topic, this line is present referring to the 3rd party logging providers ...

> For more information, see each framework's documentation.

The idea on this issue is to make that read ...

> For more information, see each provider's documentation. Third-party logging providers aren't supported by Microsoft.
